lib/spack/spack/cmd/buildcache.py

```diff
@@ -399,9 +399,13 @@ def _createtarball(env, spec_yaml=None, packages=None, add_spec=True,
 
     for spec in specs:
         tty.debug('creating binary cache file for package %s ' % spec.format())
-        bindist.build_tarball(spec, outdir, force, make_relative,
-                              unsigned, allow_root, signing_key,
-                              rebuild_index)
+        try:
+            bindist.build_tarball(spec, outdir, force, make_relative,
+                                  unsigned, allow_root, signing_key,
+                                  rebuild_index)
+        except Exception as e:
+            tty.warn('%s' % e)
+            pass
```

Comment:
```suggestion
            tty.warn(e)
```
This should be equivalent.